packages/cozy-client/src/promise-cache.js

```diff
@@ -41,6 +41,17 @@ class PromiseCache {
       this.pending[key] = null
     }
   }
+  /**
+   *
+   * @param {function(): string} keyFunc - Returns a key to find in cache to find a pending promise.
+   * @returns {Promise | null}
+   */
+  get(keyFunc) {
+    const key = keyFunc()
+    const already = this.pending[key]
+    if (already) return already
+    return null
```

Comment:
nit: in my (strictly personal) point of view `return already || null` would be clearer. But it's just a matter of code style.